server/Mac/mf_peer.c

```diff
@@ -229,8 +229,8 @@ void mf_peer_context_free(freerdp_peer* client, mfPeerContext* context)
 void mf_peer_init(freerdp_peer* client)
 {
 	client->ContextSize = sizeof(mfPeerContext);
-	client->ContextNew = mf_peer_context_new;
-	client->ContextFree = mf_peer_context_free;
+	client->ContextNew = (psPeerContextNew)mf_peer_context_new;

```

Comment:
Why did you revert that? If you have warnings your function signatures do not match.